wpilibj/src/main/java/edu/wpi/first/wpilibj/SensorUtil.java

```diff
@@ -17,64 +17,61 @@
 import edu.wpi.first.wpilibj.hal.SolenoidJNI;
 
 /**
- * Base class for all sensors. Stores most recent status information as well as containing utility
- * functions for checking channels and error processing.
+ * Stores most recent status information as well as containing utility functions for checking
+ * channels and error processing.
  */
-public abstract class SensorBase extends SendableBase {
+public class SensorUtil {
   /**
    * Ticks per microsecond.
    */
-  public static final int kSystemClockTicksPerMicrosecond =
-      ConstantsJNI.getSystemClockTicksPerMicrosecond();
+  public static final int kSystemClockTicksPerMicrosecond
+      = ConstantsJNI.getSystemClockTicksPerMicrosecond();
+
   /**
    * Number of digital channels per roboRIO.
    */
   public static final int kDigitalChannels = PortsJNI.getNumDigitalChannels();
+
   /**
    * Number of analog input channels per roboRIO.
    */
   public static final int kAnalogInputChannels = PortsJNI.getNumAnalogInputs();
+
   /**
    * Number of analog output channels per roboRIO.
    */
   public static final int kAnalogOutputChannels = PortsJNI.getNumAnalogOutputs();
+
   /**
    * Number of solenoid channels per module.
    */
   public static final int kSolenoidChannels = PortsJNI.getNumSolenoidChannels();
+
   /**
    * Number of PWM channels per roboRIO.
    */
   public static final int kPwmChannels = PortsJNI.getNumPWMChannels();
+
   /**
    * Number of relay channels per roboRIO.
    */
   public static final int kRelayChannels = PortsJNI.getNumRelayHeaders();
+
   /**
    * Number of power distribution channels per PDP.
    */
   public static final int kPDPChannels = PortsJNI.getNumPDPChannels();
+
   /**
    * Number of power distribution modules per PDP.
    */
   public static final int kPDPModules = PortsJNI.getNumPDPModules();
+
   /**
    * Number of PCM Modules.
    */
   public static final int kPCMModules = PortsJNI.getNumPCMModules();
 
-  private static int m_defaultSolenoidModule = 0;
```

Comment:
What’s the reason for removing the default solenoid module setter, but keeping the getter?

Reply:
It's easier to update the default solenoid module number in one place than three.
```
[tav@myriad allwpilib]$ grep getDefaultSolenoidModule
wpilibj/src/main/java/edu/wpi/first/wpilibj/Compressor.java
52:    this(SensorUtil.getDefaultSolenoidModule());

wpilibj/src/main/java/edu/wpi/first/wpilibj/SensorUtil.java
230:  public static int getDefaultSolenoidModule() {

wpilibj/src/main/java/edu/wpi/first/wpilibj/Solenoid.java
31:    this(SensorUtil.getDefaultSolenoidModule(), channel);

wpilibj/src/main/java/edu/wpi/first/wpilibj/DoubleSolenoid.java
43:    this(SensorUtil.getDefaultSolenoidModule(), forwardChannel, reverseChannel);
```

Reply:
Is it cleaner to leave the constant and have the getter get it?

Reply:
I don't think that adds any extra information. It's obvious what that number being returned is based on the name of the function.

Reply:
But we're getting rid of the functionality of *setting* the default.  Is that something we feel is necessary to do?

Reply:
Actually, C++ has never had a function to set the default solenoid module; only Java has.